onnx/defs/nn/defs.cc

```diff
@@ -495,7 +495,10 @@ computes the output.)DOC";
         "Optionally, if dimension denotation is in effect, "
         "the operation expects the weight tensor to arrive "
         "with the dimension denotation of [FILTER_IN_CHANNEL, "
-        "FILTER_OUT_CHANNEL, FILTER_SPATIAL, FILTER_SPATIAL ...].",
+        "FILTER_OUT_CHANNEL/group, FILTER_SPATIAL, FILTER_SPATIAL ...]. "
+        "Note that X.shape[1] should be equal to W.shape[1] * group "
```

Comment:
It should be [FILTER_OUT_CHANNEL, FILTER_IN_CHANNEL, FILTER_SPATIAL, ...]

And FILTER_IN_CHANNEL = DATA_CHANNEL / group

and you don't need the following note I think.

Reply:
If you look at the caffe2 implementation, X.shape[1] = W.shape[1]*group. In the terminology of the spec, this translates to DATA_CHANNEL = FILTER_OUT_CHANNEL where W.shape[1] = FILTER_OUT_CHANNEL/group.

Reply:
What's the meaning of FILTER_OUT_CHANNEL? does it mean the channel of the output?

Reply:
Yes, that's how the implementation interprets it. FILTER_OUT_CHANNEL is number of channels in the output.

Reply:
There are two problems in this case, 
1) the denotation filter should be [out, in, spatial, spatial]
2) the w.shape[1] * group == x.shpae[1] == C, so the W's shape should be [M, C/group, kH, kW]

Reply:
+1 for #2. 
Regarding #1, are you suggesting that we change the denotation as well in the spec to [FILTER_OUT_CHANNEL, FILTER_IN_CHANNEL, FILTER_SPATIAL, FILTER_SPATIAL ...] ?